ddtrace/internal/ci_visibility/writer.py

```diff
@@ -46,7 +46,7 @@ def __init__(
         processing_interval=get_writer_interval_seconds(),  # type: float
         timeout=agent.get_trace_agent_timeout(),  # type: float
         dogstatsd=None,  # type: Optional[DogStatsd]
-        sync_mode=True,  # type: bool
+        sync_mode=False,  # type: bool
```

Comment:
is this how the agent writer works? We'd want it to work the same way.

Reply:
Yes it is.